lib/extensions/ar_visible_attribute.rb

```diff
@@ -0,0 +1,23 @@
+module ArVisibleAttribute
+  extend ActiveSupport::Concern
+
+  included do
+    cattr_accessor :hidden_attribute_names
```

Comment:
think we can use the default here: `class_attribute :hidden_attribute_names, default: []`